sandbox/settings.py

```diff
@@ -288,6 +288,7 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
 
     CMS_TEMPLATES = (
         ("courses/cms/course_detail.html", _("Course page")),
+        ("courses/cms/course_run.html", _("Course run page")),
```

Comment:
Not sure how a course run page is usefull, but it should be called course_run_detail.html, 
(or did you mean course_runs.html ?)

Reply:
I think i'll go with course_run_detail.html